db/migrate/20150818142457_add_vm_template_id_to_vms.rb

```diff
@@ -0,0 +1,5 @@
+class AddVmTemplateIdToVms < ActiveRecord::Migration
+  def change
+    add_column :vms, :vm_template_id, :bigint
```

Comment:
@chessbyte @blomquisg  @Fryguy 

cool, seems like it's stored in vm.genealogy_parent

Is there suppose to be some UI for that?, or should I keep UI part of this patch?